lib/modules/login/login_interactor.dart

```diff
@@ -15,13 +15,11 @@ class LoginInteractorImpl extends LoginInteractor {
 
   @override
   void login(String email, String password) {
-    Future.microtask(() async {
-      try {
-        await _authRepository.login(email: email, password: password);
-        delegate?.didLogin.add(null);
-      } on Exception catch (e) {
-        delegate?.didFailToLogin.add(e);
-      }
-    });
+    _authRepository
+        .login(email: email, password: password)
+        .then((value) => null)
```

Comment:
can't we just add the `.then((value) => delegate?.didLogin.add(null))` and remove this? 🤔

Reply:
@markgravity I left a question here 5 days ago 👋

Reply:
@sleepylee let me fix it 😁